.gitignore

```diff
@@ -0,0 +1,3 @@
+/mvnw
+/mvnw.cmd
+/.mvn/wrapper/*
```

Comment:
I see all those are tracked by git, shouldn't this gitignore be removed?

Reply:
Yes! This is a leftover of a previous iteration. Thanks!